spec/models/spree/calculator/price_sack_spec.rb

```diff
@@ -16,13 +23,23 @@
     it "uses the preferred normal amount" do
       expect(calculator.compute(line_item)).to eq(10)
     end
+    context "and preferred normal amount is float" do
```

Comment:
instead of floatCalculator I think you can just do here:
before do
calculator.update_attribute :preferred_normal_amount, 10.4
end

Reply:
Hmm... doing this seems to be giving the error "PG::NotNullViolation: ERROR:  null value in column "calculable_id" violates not-null constraint".

Reply:
oh, that's probably because the object is not ready to be persisted (update_attribute persists the object).
so you could try in memory:
before do
calculator.preferred_normal_amount = 10.4
end